src/maliit-plugins.prf.in

```diff
@@ -1,3 +0,0 @@
-LIBS *= -L@LIBDIR@ -l@MALIIT_PLUGINS_LIB@
```

Comment:
Removing the offered .prf files affect also the depending projects. Suggest not doing it.